src/main/java/com/vinylteam/vinyl/dao/jdbc/JdbcUserDao.java

```diff
@@ -52,36 +58,43 @@ public boolean add(User user) {
         try (Connection connection = DBDataSource.getConnection();
              PreparedStatement findByEmailStatement = connection.prepareStatement(FIND_BY_EMAIL)) {
             findByEmailStatement.setString(1, email);
+            logger.debug("Prepared statement {'preparedStatement':{}}.", findByEmailStatement);
             try (ResultSet resultSet = findByEmailStatement.executeQuery()) {
+                logger.debug("Executed statement with result set {'preparedStatement':{}, 'resultSet':{}}",
+                        findByEmailStatement, resultSet);
                 if (resultSet.next()) {
                     user = userRowMapper.mapRow(resultSet);
                     if (resultSet.next()) {
-                        logger.error("More than one user was found for email: {}", email);
-                        throw new RuntimeException("More than one user was found for email: ".concat(email));
+                        RuntimeException e = new RuntimeException();
+                        logger.error("More than one user was found for email: {}", email, e);
+                        throw e;
                     }
                 }
             }
         } catch (SQLException e) {
             logger.error("SQLException retrieving user by email from public.users", e);
             throw new RuntimeException(e);
         }
+        logger.debug("Resulting optional with user is {'Optional.ofNullable(user)':{}}",
+                Optional.ofNullable(user));
         return Optional.ofNullable(user);
     }
 
     int countAll() {
-
         int count = -1;
         try (Connection connection = DBDataSource.getConnection();
              Statement countStatement = connection.createStatement();
              ResultSet resultSet = countStatement.executeQuery(COUNT_ALL)) {
+            logger.debug("Executed statement with result set {'statement':{}, 'resultSet':{}}",
+                    countStatement, resultSet);
```

Comment:
What info about resultSet we can see in logs after ResultSet.toString()? If nothing interesting please remove it from log and leave only log of statement in one row, thanks.